paper.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Qiskit Dynamics: A Python package for simulating the time dynamics of quantum systems'
+tags:
+  - Python
+  - quantum
+  - quantum computer
+  - pulse
+  - control
+authors:
+  - name: Daniel Puzzuoli
+    corresponding: true
+    affiliation: 1
+  - name: Christopher J. Wood
+    affiliation: 2
+  - name: Daniel J. Egger
+    affiliation: 3
+  - name: Benjamin Rosand
+    affiliation: 4
+  - name: Kento Ueda
+    affiliation: 5
+  - name: Ian Hincks
+    affiliation: 6
+  - name: Haggai Landa
+    affiliation: 7
+  - name: Moein Malekakhlagh
+    affiliation: 2
+  - name: Avery Parr
+    affiliation: 8
+  - name: R. K. Rupesh
+    affiliation: 9
+  - name: Matthew Treinish
+    affiliation: 2
+affiliations:
+ - name: IBM Quantum, IBM Canada, Vancouver, BC, Canada
+   index: 1
+ - name: IBM Quantum, IBM T.J. Watson Research Center, Yorktown Heights, NY, USA
+   index: 2
+ - name: IBM Quantum, IBM Research Europe - Zurich, Ruschlikon, Switzerland
+   index: 3
+ - name: Department of Physics, Yale University, New Haven, CT, USA
+   index: 4
+ - name: IBM Quantum, IBM Research Tokyo, Tokyo, Japan
+   index: 5
+ - name: IBM Quantum, IBM Canada, Markham, ON, Canada
+   index: 6
+ - name: IBM Quantum, IBM Research Israel, Haifa, Israel
+   index: 7
+ - name: Department of Physics, Harvard University, Cambridge, MA, USA
+   index: 8
+ - name: School of Physics, University of Hyderabad, Hyderabad, India
+   index: 9
+
+
+date: 31 July 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+Qiskit Dynamics is an open source Python library for numerically simulating the time dynamics of finite-dimensional quantum systems. The goal of the package is to provide flexible configuration of the numerical methods used for simulation: general tools for transforming models of quantum systems for more efficient simulation (rotating frames and the rotating wave approximation), choice of array representations (dense v.s. sparse, and different array libraries), and access to different types of underlying solvers (standard ODE v.s. geometric solvers). The package also contains advanced functionality for computing time-dependent perturbation theory expressions used in robust quantum control optimization `[@perturb1; @perturb2]`. 
```

Comment:
I wonder if this summary is a too heavy on the jargon? E.g. rotating frame
```suggestion
Qiskit Dynamics is an open source Python library to numerically simulate the time dynamics of finite-dimensional quantum systems. The package provides a flexible configuration of the numerical methods used for the simulation. Users can chose how to transform the models of the quantum systems to enable efficient simulations, how to represent the underlying arrays (e.g., dense v.s. sparse with different array libraries), and with which solvers to time-evolve the models (e.g., standard ODE v.s. geometric solvers). Qiskit Dynamics also contains advanced functionality to compute time-dependent perturbation theory expressions used in robust quantum control optimization `[@perturb1; @perturb2]`. 
```

Reply:
Yeah I was on the fence about including "rotating frame" in the text. As a compromise I ended up putting it in parentheses, as I'd still like to be specific for readers that know what these things are.